.github/workflows/docs-checker.yml

```
@@ -0,0 +1,25 @@
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened, edited ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: 'Setup Environment'
+        uses: actions/setup-python@v2
```

actionlint (1.7.4)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)